fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -2748,7 +2748,7 @@ class DDTeamCollectionImpl {
 					when(wait(checkSignal)) {
 						checkSignal = Never();
 						isFetchingResults = true;
-						serverListAndProcessClasses = getServerListAndProcessClasses(&tr);
+						serverListAndProcessClasses = NativeAPI::getServerListAndProcessClasses(&tr);
```

Comment:
This should go through the transaction processor, shouldn't it?

Reply:
Yes. That's the goal. Last week I just want to make each PR small so I leave it to call NativeAPI now. A future PR will do the switch.